src/validations/test/rules/poam.xspec

```diff
@@ -8,29 +8,329 @@
+
+
+            <x:scenario
+                label="when the import-ssp element">
+
+                <x:scenario
+                    label="is present">
+                    <x:expect-not-assert
+                        id="has-import-ssp"
+                        label="that is correct" />
+                </x:scenario>
+
+                <x:scenario
+                    label="is absent">
+                    <x:context>
+                        <assessment-plan
+                            xmlns="http://csrc.nist.gov/ns/oscal/1.0">
+                            <!-- missing import-ssp -->
+                        </assessment-plan>
+                    </x:context>
+                    <x:expect-assert
+                        id="has-import-ssp"
+                        label="that is an error" />
+                </x:scenario>
+
+                <x:scenario
+                    label="has an href attribute">
+                    <x:expect-not-assert
+                        id="has-import-ssp-href"
+                        label="that is correct" />
+                </x:scenario>
+
+                <x:scenario
+                    label="lacks an href attribute">
+                    <x:context>
+                        <assessment-plan
+                            xmlns="http://csrc.nist.gov/ns/oscal/1.0">
+                            <import-ssp /><!-- missing href -->
+                        </assessment-plan>
+                    </x:context>
+                    <x:expect-assert
+                        id="has-import-ssp-href"
+                        label="that is an error" />
+                </x:scenario>
+
+            </x:scenario>
+
+            <x:scenario
+                label="when the import-ssp has a relative href">
+                <x:scenario
+                    label="which refers to a resource">
+                    <x:expect-not-assert
+                        id="has-import-ssp-internal-href"
+                        label="that is correct" />
+                </x:scenario>
+                <x:scenario
+                    label="which does not refer to a resource">
+                    <x:context>
+                        <assessment-plan
+                            xmlns="http://csrc.nist.gov/ns/oscal/1.0">
+                            <import-ssp
+                                href="#bf974aa0-d662-4aef-a7a5-8f4cce398746" />
+                            <back-matter>
+                                <!-- missing resource -->
+                            </back-matter>
+                        </assessment-plan>
+                    </x:context>
+                    <x:expect-assert
+                        id="has-import-ssp-internal-href"
+                        label="that is an error" />
+                </x:scenario>
+            </x:scenario>
+
+            <x:scenario
+                label="when the import-ssp has an external href"
+                pending="XSpec problem resolution">
```

Comment:
Did you intend to keep this scenario pending?

Reply:
Yes. I cannot find the associated XSpec error. The target of the href is not reachable. However, I removed the subsequent one.